iOS/Flows/Circles/CircleCollectionViewLayout.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  CircleCollectionViewLayout.swift
+//  Jibber
+//
+//  Created by Benji Dodgson on 1/26/22.
+//  Copyright © 2022 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+
+class CircleCollectionViewLayout: UICollectionViewLayout {
+    
+    // MARK: Getting the Collection View Information
+    
+    override var collectionViewContentSize: CGSize {
```

Comment:
This still needs some love. Would be nice to scroll horizontally so you could actually see everything.